drivers/wifi/nxp/Kconfig.nxp

```diff
@@ -316,7 +316,8 @@ menu "Wi-Fi driver Stack configurations"
 
 config NXP_WIFI_MON_TASK_STACK_SIZE
 	int "Mon thread stack size"
-	default 1152
+	depends on NXP_RW610
```

Comment:
Do we only need the `mon` task on the RW610?

Reply:
Yes, the mon task is only used for RW610 to handle host sleep and wifi recovery in Wi-Fi driver.